privacyidea/static/contrib/css/animate.css

```diff
@@ -22,3 +22,27 @@
           transform: scaleX(1);
   opacity: 1;
 }
+.spin {
```

Comment:
An animated gif? 😮 
CSS has come a long way!

I will attempt to reproduce the problem on a different environment.